README.md

```diff
@@ -55,294 +53,16 @@ Also, the Pattern Library currently relies on the following thrd party libraries
 
 ## Documentation
 
-The UX Pattern Library has its own living documentation site at
+The edX Pattern Library has its own living documentation site at
 http://ux.edx.org. Additionally, we have many [styleguides and how-to wiki
 documents](https://github.com/edx/ux-pattern-library/wiki) in the Github
 repository.
 
-## Development - Documentation Site (PLDOC)
+## How to Develop & Use
+* If you want to use the edX Pattern Library in your project or work, please see [How to Work and Develop on Documentation Site (PLDOC)](https://github.com/edx/ux-pattern-library/wiki#how-to-use-and-deploy-the-uxpl-in-your-project)
+* If you'd like to set up or work on the living documentation site locally, check out [How to Work and Develop on Documentation Site (PLDOC)](https://github.com/edx/ux-pattern-library/wiki#how-to-work-and-develop-on-documentation-site-pldoc)
+* If you'd like to work on or contribute to the edX Pattern Library package, read [How to Work/Develop on the edX Pattern Library Package](https://github.com/edx/ux-pattern-library/wiki#how-to-workdevelop-on-the-ux-pattern-library-package)
 
-To work on the code of the documentation site portion of this repo, you'll need
```

Comment:
Sweet! It's great to remove all of this content. It will make it much easier for folks to navigate.